frontend/appflowy_flutter/lib/user/presentation/screens/sign_in_screen/widgets/continue_with/continue_with_email_and_password.dart

```diff
@@ -38,16 +37,12 @@ class _ContinueWithEmailAndPasswordState
 
     return Column(
       children: [
-        SizedBox(
-          height: UniversalPlatform.isMobile ? 38.0 : 40.0,
-          child: AFTextField(
-            controller: controller,
```

Comment:
@LucasXu0 Do we have a separate spec for mobile?

Reply:
I've deleted them.

https://github.com/AppFlowy-IO/AppFlowy/pull/7730/files